superset/charts/data/commands/get_data_command.py

```diff
@@ -14,6 +14,8 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from __future__ import annotations
+
```

Comment:
I develop with py 3.8, without it raises errors

Reply:
Yep, let's remove this if we can.

Reply:
https://docs.python.org/3/library/__future__.html

![image](https://user-images.githubusercontent.com/35701650/144020171-d5ecaa5e-c8df-4aac-8499-8455b071f98f.png)

Reply:
@villebro 
regarding the FF, I don't agree with it since the change is fixing a bug a not adding new functionality.
if someone will encounter a surprising issue with it, it means "hey it got something, we have a security breach" so the app administrator should think about how to solve it, giving extra permission or removing the metric.

this is my IMO, but for now I'll add that FF

Reply:
@ofekisr My point for introducing the FF is to gradually introduce the new stricter security check in a way which makes it possible to catch potential regressions that weren't caught during initial review. By having an introductory period during which the stricter checks are behind a FF, orgs that require the new stricter checking can help harden the feature by pushing fixes to master before the new checks are made generally available.